com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -913,6 +918,18 @@ public bool useAdaptivePerformance
             set { m_UseAdaptivePerformance = value; }
         }
 
+        public bool conservativeEnclosingSphere
+        {
+            get { return m_ConservativeEnclosingSphere; }
+            set { m_ConservativeEnclosingSphere = value; }
+        }
+
+        public int numItertionsEnclosingSphere
```

Comment:
A bit late to the party but there's a small spelling mistake here:

numItertionsEnclosingSphere -> numIterationsEnclosingSphere